_data/comments/2016-08-10-adding-user-generated-content-to-a-static-site-using-staticman/entry1473777412967.yml

```diff
@@ -0,0 +1,8 @@
+name: Ivan
```

Comment:
@eduardoboucas 
Wow, it works! I'll make another attempt to use it. Encouraging block, thanks a lot!